conductor/server/entrypoint/start_conductor.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+/app/startup.sh &
+sleep 60
+
+# load workflow definitions
+shopt -s nullglob
+for workflow in /srv/workflows/*.json; do
+    curl -X POST -H "Content-Type: application/json" http://localhost:8080/api/metadata/workflow -d @$workflow
```

Comment:
please note 2 things:
* if a given workflow already exists, 409 conflict will be returned - do we want to handle this?
* workflows a re uploaded in 2 step, first task definitions, then workflow definitions, see e.g.:
https://tracker.mender.io/browse/MEN-1097?focusedCommentId=78777&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-78777